src/app/app.component.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Climate Change Lab
+ * App Component
+ */
+import { Component, ViewEncapsulation } from '@angular/core';
+
+/*
+ * App Component
+ * Top Level Component
+ */
+@Component({
+  selector: 'app',
+  encapsulation: ViewEncapsulation.None,
+  styleUrls: [
+    './app.style.css'
+  ],
+  templateUrl: './app.template.html'
+})
+export class App {
+  name = 'Climate Lab';
+
+  ngOnInit() {
```

Comment:
Remove. This requires the class to implement the OnInit interface to work anyways.